ArmyOfOnes/ArmyOfOnes/ViewControllers/CurrenciesListViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  CurrenciesListViewController.swift
+//  ArmyOfOnes
+//
+//  Created by Fabian Mariño on 11/8/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+enum LayoutConstraints: CGFloat {
+    case inputOffset = 14,
+    topInset = 43,
+    tableTopInset = 70
+}
+
+enum textFieldLimit: Int {
+    case max = 15
+}
+
+class CurrenciesListViewController: UITableViewController {
```

Comment:
I'd advice to avoid using `UITableViewController`. It limits you a lot with what you can do with that view controller. It's generally better to use just a `UIViewController` and add a `UITableView` as a subview